azurelinuxagent/common/utils/fileutil.py

```diff
@@ -123,6 +124,15 @@ def chmod(path, mode):
         os.chmod(path, mode)
 
 
+def umount(*args):
+    for paths in args:
+        # find all possible mounts
+            for path in glob.glob(paths):
```

Comment:
Unnecessary indentation here after the comment.

Reply:
Yep, the indentation is unnecessary here. I'll amend that.